VideoWeb/VideoWeb/ClientApp/src/app/shared/header-logo-svg/header-logo-svg.component.html

```diff
@@ -1,4 +1,6 @@
-<svg role="presentation" focusable="false" class="moj-header__logotype-crest" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" height="40" width="40">
+<img role="presentation" id="header-logo-scot" class="moj-header__logotype-crest" *ngIf="hearingVenueIsInScotland" src="/assets/images/scot-crest-white.png" height="40" [attr.alt]="'participants-panel.scottish-crest' | translate"/>
+
+<svg id="header-logo-uk" *ngIf="!hearingVenueIsInScotland" role="presentation" focusable="false" class="moj-header__logotype-crest" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" height="40" width="40">
```

Comment:
I don't think we should put the img with png inside svg component. Maybe do this check in the header? Also if you have *ngIf for a true and false I would recommend using ng-template with *ngIf then else
https://ultimatecourses.com/blog/angular-ngif-else-then

Reply:
yea I will move it out of the svg component

Reply:
argh, also appears I have missed converting it into using an async pipe anyways